deepinv/loss/measplit.py

```diff
@@ -108,6 +108,7 @@ def forward(self, x_net, y, physics, model, **kwargs):
         """
         mask = model.get_mask()
 
+        mask *= getattr(physics, "mask", 1.0)

```

Comment:
What is the reasoning here? If a `physics.mask` is present (e.g. in MRI), `mask` will already be subsampled from `physics.mask` (in the `SplittingModel.forward`), so this line does nothing.

Reply:
it is a safety line for the case where the second mask is not subsampled.

Reply:
I have added this in #279 now.